priv/repo/migrations/20161128200526_create_user.exs

```diff
@@ -0,0 +1,16 @@
+defmodule MovieWagerApi.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :screen_name, :string, null: false
+      add :profile_image_url, :string
+      add :twitter_id, :integer, null: false
```

Comment:
There is no existing user data yet as this is the creation step of users table. Right now, I think just Twitter is fine. I think adding other social media links is out of scope of the project.